test/automl/test_multiclass.py

```diff
@@ -3,8 +3,9 @@
 import scipy.sparse
 from sklearn.datasets import load_iris, load_wine
 from flaml import AutoML
-from flaml.automl.data import CLASSIFICATION, get_output_from_log
+from flaml.automl.data import get_output_from_log
 from flaml.automl.model import LGBMEstimator, XGBoostSklearnEstimator, SKLearnEstimator
+from flaml.automl.task.task import CLASSIFICATION
```

Comment:
Is line 18 still correct when task is a str?

Reply:
I've fixed it now, thanks